mirgecom/eos.py

```diff
@@ -531,6 +531,95 @@ def get_internal_energy(self, temperature, species_mass_fractions=None):
         """
         return self._gas_const * temperature / (self._gamma - 1)
 
+    def conservative_to_entropy_vars(self, cv, temperature):
+        """Compute the entropy variables from conserved variables.
+
+        Converts from conserved variables (density, momentum, total energy)
+        into entropy variables.
```

Comment:
Maybe link to the definition of the entropy variables, if they're documented somewhere?

Reply:
They aren't to my knowledge, but this PR should add it, drawing entirely or at least heavily from the esdg writeup from @thomasgibson.